eng/helix/content/RunTests/TestRunner.cs

```diff
@@ -55,9 +55,6 @@ public bool SetupEnvironment()
                 var playwrightBrowsers = Environment.GetEnvironmentVariable("PLAYWRIGHT_BROWSERS_PATH");
                 Console.WriteLine($"Setting PLAYWRIGHT_BROWSERS_PATH: {playwrightBrowsers}");
                 EnvironmentVariables.Add("PLAYWRIGHT_BROWSERS_PATH", playwrightBrowsers);
-                var playrightDriver = Environment.GetEnvironmentVariable("PLAYWRIGHT_DRIVER_PATH");
```

Comment:
Doesn't seem to be possible to specify driver path any longer:

https://github.com/microsoft/playwright-dotnet/blob/45de47f3f51c086278b347e09a878731d996e2e5/src/Playwright/Helpers/Paths.cs